quote_test.py

```diff
@@ -1,13 +1,32 @@
 import pytest
 from quote import QuoteParagraph
 
-@pytest.mark.parametrize(
-    "content, html",
-    [
-        ('abc', "<blockquote> abc </blockquote>"),
-        ('<html> rand tag </html>', "<blockquote> <html> rand tag </html> </blockquote>"),
-    ]
-)
+@pytest.mark.parametrize("content, html", [
+    (r'abc', r"<blockquote>abc</blockquote>"),
+    (r'<html> rand tag </html>',
+     r"<blockquote><html> rand tag </html></blockquote>"),
+])
 def test_render(content, html):
-    quote = QuoteParagraph(content)
+    _, _, quote = QuoteParagraph(content)
```

Comment:
We should keep the previous instantiation

Reply:
done